zerver/tests/fixtures/ldap/directory.ldif

```diff
@@ -0,0 +1,89 @@
+dn: dc=zulip,dc=com
```

Comment:
Let's add a comment at the top explaining that this is the test LDAP database used in the development environment.